index.js

```diff
@@ -12,6 +12,14 @@ const deepEqSkip = function(data, expectedData, skipKeys = []) {
   if (!isObj(data)) {
     return (data === expectedData);
   }
+
+  // If data is empty then throw an error
```

Comment:
We do not want to just check the empty condition.
Here we are matching data with expectedData so I think there should be following this conditions
```
- If data is empty and expectedData is not empty
    - then throw new Error(`Expected ${JSON.strigify(expectedData)} found ${JSON.stringify(data)}`)
- If data is not empty and expectedData is empty
   - then throw new Error(`Expected ${JSON.strigify(expectedData)} found ${JSON.stringify(data)}`)
```